benchmarks/devel/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_executable(ArborX_Benchmark_devel.exe devel.cpp)
+target_link_libraries(ArborX_Benchmark_devel.exe ArborX::ArborX benchmark::benchmark)
+add_test(NAME ArborX_Benchmark_Dummy COMMAND ./ArborX_Benchmark_devel.exe)
```

Comment:
```suggestion
add_executable(ArborX_Benchmark_Devel.exe devel.cpp)
target_link_libraries(ArborX_Benchmark_Devel.exe ArborX::ArborX benchmark::benchmark)
add_test(NAME ArborX_Benchmark_Devel COMMAND ./ArborX_Benchmark_Devel.exe)
```

Reply:
Hoped you would be ok with `devel`, I know `Devel` is our style but it just looks weird do me, so used to `devel` everywhere.

Reply:
Except the one "dummy" you missed.  What looks weird, our style or just this one?

Reply:
Just `Devel` with capital `D`. So used to 1.2.3-devel.